src/Engine/PriceCalculator.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+#include "Engine/Objects/Items.h"
+
+class Player;
+// Class to calculate various prices and overall deal with them
+class PriceCalculator {
+ public:
+  static int getBaseIdentifyPrice(float priceMultiplier);
+  static int getBaseRepairPrice(int uRealValue, float priceMultiplier);
+  static int getBaseBuyingPrice(int uRealValue, float priceMultiplier);
+  static int getBaseSellingPrice(int uRealValue, float priceMultiplier);
+
+  static int getItemRepairPriceForPlayer(Player* player, int uRealValue,
+                                         float priceMultiplier);
```

Comment:
Same thing about the line length – 120 chars is perfectly OK, 160 is the comfortable limit.